Dockerfile

```diff
@@ -138,6 +145,31 @@ ENV DEBIAN_FRONTEND=noninteractive \
     RCON_PORT=25575 \
     REGION= \
     USEAUTH=true \
-    BAN_LIST_URL=https://api.palworldgame.com/api/banlist.txt
+    BAN_LIST_URL=https://api.palworldgame.com/api/banlist.txt \
+    ### Webhook-settings
```

Comment:
This is a container-setting, it belongs up after the SERVER_SETTINGS, that was on purpose, no need to make readibility less then good.

Move to below SERVER_SETTINGS_MODE

Reply:
Actually above would be better as then server settings would be right next to the settings themselves, but can move up sure.